tests/test_qaoa_objective_sk.py

```diff
@@ -0,0 +1,213 @@
+###############################################################################
+# // SPDX-License-Identifier: Apache-2.0
+# // Copyright : JP Morgan Chase & Co
+###############################################################################
+import numpy as np
+import pytest
+from pathlib import Path
+from functools import partial
+from qiskit_aer import AerSimulator
+from itertools import combinations
+from qokit.sk import sk_obj, get_sk_terms
+from qokit.maxcut import maxcut_obj
+from qokit.qaoa_objective_sk import get_qaoa_sk_objective
+from qokit.utils import brute_force, precompute_energies
+from qokit.parameter_utils import get_sk_gamma_beta
+from qokit.fur import get_available_simulators, get_available_simulator_names
+from qokit.qaoa_objective import get_qaoa_objective
+
+test_sk_folder = Path(__file__).parent
+
+np.random.seed(100)
+
+qiskit_backend = AerSimulator(method="statevector")
+SIMULATORS = get_available_simulators("x") + get_available_simulators("xyring") + get_available_simulators("xycomplete")
+simulators_to_run_names = get_available_simulator_names("x") + ["qiskit"]
+simulators_to_run_names_no_qiskit = get_available_simulator_names("x")
```

Comment:
In all of the below tests qiskit is excluded (only `simulators_to_run_names_no_qiskit` is used). Why?